webrtc.html

```diff
@@ -383,8 +383,8 @@
           </li>
 
           <li>
-            <p>Initialize an internal variable to represent a queue of
-            <code>operations</code> with an empty set.</p>
+            <p>Initialize an internal variable to a resolved promise, to
+            represent the start of a chain of <code>operations</code>.</p>
```

Comment:
I believe that it's customary to talk about this as "an internal slot called [[operations]]"